modules/nf-core/crumble/meta.yml

```diff
@@ -0,0 +1,59 @@
+name: "crumble"
+description: Controllable lossy compression of BAM/CRAM files
+keywords:
+  - compress
+  - bam
+  - sam
+  - cram
+tools:
+  - "crumble":
+      description: "Controllable lossy compression of BAM/CRAM files"
+      homepage: "https://github.com/jkbonfield/crumble"
+      documentation: "https://github.com/jkbonfield/crumble"
+      tool_dev_url: "https://github.com/jkbonfield/crumble"
+      doi: "10.1093/bioinformatics/bty608"
+      licence: "['multiple BSD style licenses']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - input:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+  - bed:
+      type: boolean
+      description: Specify if suspicious regions need to be output in BED format
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - bam:
+      type: file
+      description: optional filtered/compressed BAM file
+      pattern: "*.{bam}"
+  - cram:
+      type: file
+      description: optional filtered/compressed CRAM file
+      pattern: "*.{cram}"
+  - sam:
+      type: file
+      description: optional filtered/compressed SAM file
+      pattern: "*.{sam}"
+  - bed:
+      type: file
+      description: optional suspicious regions BED file
```

Comment:
Maybe specify here what argument the user should use to get this BED file

Reply:
Updated